incuna-sass/mixins/_svg.sass

```diff
@@ -0,0 +1,12 @@
+//////////  S V G  M I X I N  //////////////////////////////
+// To use like you would a sprite with the added benefit that
+// it can be whatever size you like and will work perfectly
+// on retina screens
+@mixin svg($file, $width: 100%, $height: $width)
+    @include background(image-url('svg/#{$file}.svg'))
+    background:
+        repeat: no-repeat
+        size: contain
+    width: $width
+    height: $height
+    display: inline-block
```

Comment:
>how is having two mixins that do exactly the same thing only doing as much as possible?

It was an attempt to satisfy your need of having the display set =) Sorry, I should have said:

>I prefer library things like `@mixin svg` to do only as much as is necessary

It doesn't _need_ to set the display, even if we can imagine that `inline-block` would be the majority usage of it. Ideally it should use the element's default display.

Here's a comparison:

If display is set here:
```sass
.my-block-level-element
    @include svg('my-shape')
    display: block
    @include media($larger)
        @include svg('my-shape-larger')
        // We have to set display again.
        display: block
    @include media($even-larger)
        @include svg('my-shape-even-larger')
        // We have to set display again.
        display: block
```

If display is not set here:
```sass
.my-icon
    @include svg('my-icon')
    // We have to set display here to make it inline.
    display: inline-block
    // But...
    @include media($larger)
        @include svg('my-icon-larger')
        // We don't have to set display again.
    @include media($even-larger)
        @include svg('my-icon-even-larger')
        // We don't have to set display again.
```